### PR TITLE
Add Set-NSXLoadBalancerPool

### PR DESCRIPTION
**Pull request by alagoutte**
for modified settings (Name, Description, Transparent Mode, Algorithm) of Load Balancer Pool

Missing to modified Monitor Profile

### CODE REVIEW

**Comment by alagoutte** [comment]
I will update the patch to support last repo change..
What the best for easy development ? (actually i clone directly powernsx folder on modules...)

About Monitor Profile, the problem is for remove this setting add a Remove-NSX-EdgeMonitorProfile ?

**Comment by nmbradford** [comment]
Thanks Alexis.  Recommend method for development is fork, develop your changes in a feature branch in your fork and then PR when complete.  Can you please add tests that test any new functionality that you add as a part of the PR?  The latest repo change is fairly easy to deal with - changes are now in module/PowerNSX.psm1 and the additional function name should be added to Include.ps1.

**Comment by alagoutte** [comment]
I plan to add test but there is actually not Test for LB... (work in progress...)

My issue is about manifest file (PowerNSX.pdsl1), it is needed for the module.. and it is say for don't modified directly manifest file...

**Comment by nmbradford** [comment]
referring to the tests - PRs gratefully accepted!  referring to Manifest issues, lets discuss that in the issue related to it.

**Comment by alagoutte** [comment]
Update the patch :)

**Comment by alagoutte** [comment]
@dcoghlan adding some tests for Set-NSXLoadBalancerPool (now there is a LB Test) can you launch test suite ?

**Comment by dcoghlan** [comment]
jenkins test this please

File: tests/integration/20.LB.Tests.ps1 [hunk 1]
```diff
@@ -77,6 +77,15 @@ Describe "Edge Load Balancer" {
             $lb_pool.member[1].name | Should be "VM02"
         }
 
+        it "Configure LB Pool" {
+            #Configure Pool to LB
+            Get-NsxEdge $lbedge1name | Get-NsxLoadBalancer | Get-NsxLoadBalancerPool -name pester_lb_pool1 | Set-NsxLoadBalancerPool -name pester_lb_pool3 -Description "Pester LB Pool 3" -Transparent:$true -Algorithm ip-hash
+            $lb_pool = Get-NsxEdge $lbedge1name | Get-NsxLoadBalancer | Get-NsxLoadBalancerPool pester_lb_pool3
```
Comment (dcoghlan) [inline tests/integration/20.LB.Tests.ps1]: The tests failed due to line 112 where you only clean up 2 of the 3 lb pools. You need to make each test be independent of each other.

Maybe need to look to leverage the beforeeach and aftereach code blocks to create/cleanup the environment before/after each test.

**Comment by alagoutte** [comment]
Can you relaunch test ?

**Comment by dcoghlan** [comment]
jenkins test this please

**Comment by dcoghlan** [comment]
jenkins test this please

**Pull request merged**